bin/__tests__/getIntegrationAccessToken.test.js

```diff
@@ -232,10 +217,33 @@ describe('getIntegrationAccessToken', () => {
           metaScopes: ['https://scope.com/s/ent_reactor_admin_sdk'],
         })
       );
+
       expect(mockAuth.calls.count()).toBe(METASCOPES.length);
       // This tests that if all metascopes fail, the error from the last attempt is ultimately thrown.
       expect(errorMessage).toBe(
-        'Error retrieving access token. Invalid metascope.'
+        `Error retrieving access token. ${mockedAuthError}`
+      );
+    });
+
+    it('shows JS error details in case they happen', async () => {
+      const mockedAuthError = 'some error';
+      mockAuth.and.returnValue(
+        Promise.reject(new Error(mockedAuthError))
+      );
+
+      let errorMessage;
+      try {
+        await getIntegrationAccessToken(getEnvConfig(), getArguments());
+      } catch (error) {
+        errorMessage = error.message;
+      }
+
+      // we bailed after the first call because it wasn't a scoping error
+      expect(mockAuth.calls.count()).toBe(1)
+
+      // This tests that if all metascopes fail, the error from the last attempt is ultimately thrown.
```

Comment:
This comment doesn't apply to this test, right?

Reply:
Good catch